Resources/doc/features.md

```diff
@@ -0,0 +1,12 @@
+Features
+===========
```

Comment:
This file should removed and the two points should be moved to the mentioned places.